draft-sheffer-oauth-rfc8725bis.md

```diff
@@ -388,8 +388,14 @@ an attacker as vectors for injection attacks or server-side request forgery (SSR
 
  For mitigations, see  {{do-not-trust-claims}}.
 
+## Security Policy Is Not Defensive
 
+Some JWT libraries used a blocklist for algorithm names, but misinterpreted
```

Comment:
If we're going to use the term "blocklist", we either need to define it or add a reference for it, as it's not a commonly understood term.  I would suggest rewording to not use the term, as that may result in text that's easier to read.